pkg/exchange/binance/stream.go

```diff
@@ -80,6 +94,12 @@ func NewStream(ex *Exchange, client *binance.Client, futuresClient *futures.Clie
 		futuresClient:  futuresClient,
 		depthBuffers:   make(map[string]*depth.Buffer),
 	}
+	var privatePEM string = os.Getenv("BINANCE_STREAM_PRIVATE_PEM")
```

Comment:
btw you should load the private key pem in the Exchange constructor, in order to handle fallback to HMAC